### ISSUE TITLE
Model Training

### ISSUE THREAD

**Issue by RussellALA** [stdhd/inverse-sketcher#6]
Train the model to compare the effect of the following settings:
200 epochs each.

| Done  | Filename                    |Permutation  | ActNorm  | AE pretraining  | BatchNorm in Coupling Subnets  |
|--|----------------------|--------------|----------|----------------|-------------------------------|
| [x] |glow_v2_ae_random |  random       | movingavg   |         yes          |                   yes                         |
| [x]| glow_v2_ae_learnednorm   | soft   |            learned        |                yes   | yes  |
| [in Progress ]| glow_v2_ae_noactnorm  | soft   |            NO        |                yes   | yes  |
| [in Progress ]| glow_v2_ae_nosubnetnorm   | soft   |            movingavg        |                yes   | NO  |
| [in Progress ]| glow_v2_noae  | soft              |            movingavg        |                NO   | yes  |
